packages/fcl/src/authentication/README.md

```diff
@@ -17,7 +17,7 @@ FCL provides two ways of getting the current users information. One way is a pro
 ### Snapshot of Current User
 
 ```javascript
-import * as fcl from "@onflow/fcl"
```

Comment:
@orodio I believe the previous syntax is required. But have never actually confirmed this.

Reply:
Yeah its required

Reply:
The library was done best practices for tree shaking in mind. If people really care about tree shaking they can bring in each piece individually if they want.

```javascript
import * as fcl from "@onflow/fcl"

await fcl.send([
  fcl.script`
    pub fun main(a: Int, b: Int): Int {
      return a + b 
    }
  `,
   fcl.args([
     fcl.arg(5, t.Int),
     fcl.arg(6, t.Int),
   ])
]).then(fcl.decode)
```

can be written like:

```javascript
import {send, script, decode, args, arg} from "@onflow/fcl"
import {Int} from "@onflow/types"

await send([
  script`
    pub fun main(a: Int, b: Int): Int {
      return a + b
    }
  `,
  args([
    arg(5, Int),
    arg(6, Int),
  ])
])
```
Eventually I would expect most people to use the high level helpers though like this:
```javascript
import {query} from "@onflow/fcl"

await query({
  cadence: `
    pub fun main(a: Int, b: Int): Int {
      return a + b
    }
  `,
  args: (arg, { Int }) => [
    arg(5, Int),
    arg(6, Int),
  ]
})

```

Reply:
Oh! Sorry I thought this was outdated since some people were having issues importing these packages w/ the asterisk *